tailcfg/derpmap.go

```diff
@@ -13,7 +13,11 @@ type DERPMap struct {
 	// It's keyed by the DERPRegion.RegionID.
 	//
 	// The numbers are not necessarily contiguous.
-	Regions map[int]*DERPRegion
+	Regions map[int]*DERPRegion `json:"regions"`
```

Comment:
I think you can probably drop this change. By default Go matches either case on parse, so we can still have people write `regions` if that's prettier. But I'm worried about changing what we send to people if we change this.